order/batch_storer.go

```diff
@@ -110,6 +110,17 @@ func (s *batchStorer) StorePendingBatch(batch *Batch) error {
 				)
 			}
 
+			// The account version can be upgraded during the batch
+			// if the client supports it.
+			if batch.Version.SupportsAccountTaprootUpgrade() &&
```

Comment:
Ideally we'd add a test/assertion for this upgrade path, though I think this is already covered w/ the backend itests?

Reply:
Yes, the various upgrade paths are pretty well covered in integration tests. And also in the account manager test.